frinx-uniconfig/user-guide/uniconfig-operations/uniconfig-node-manager/rpc_checked-commit/readme.md

```diff
@@ -39,235 +39,147 @@ beginning of the transaction.
 
 ### Successful Example
 
-RPC checked-commit input has 2 target nodes and the output describes the
-result of the checked-commit.
+Nodes 'R1' and 'R2' has been changed. RPC checked-commit input invoke all
```

Comment:
Be more specific: Configuration of nodes 'R1' and 'R2' has been changed in the transaction. Both 'R1' and  'R2' are in-sync with actual state on the device.

Reply:
Done.